gradle/build-scan.gradle

```diff
@@ -0,0 +1,38 @@
+import nebula.plugin.info.scm.ScmInfoExtension
+
+buildScan {
+    // Accept Gradle ToS when project property org.elasticsearch.acceptScanTOS=true
+    if (Boolean.valueOf(project.findProperty('org.elasticsearch.acceptScanTOS') ?: "false")) {
+        termsOfServiceUrl = 'https://gradle.com/terms-of-service'
+        termsOfServiceAgree = 'yes'
+    }
+
+    // Jenkins-specific build scan metadata
+    if (System.getenv('JENKINS_URL')) {
```

Comment:
We could check if this contains `elastic.co` to make sure it's our CI and accept the terms of service if so so no additional property needs to be passed in Jenkins.

Reply:
Eventually we'll be pushing scans to Gradle Enterprise which doesn't require accepting the ToS so I think that might be unnecessary.

Reply:
I've gone ahead and implemented this. I agree, it'd be nice to ditch the need to pass the ToS param in Jenkins for the time being.